eng/pipelines/mgmt-ci.yml

```diff
@@ -1,7 +1,7 @@
 trigger:
   branches:
     include:
-      - main
+      - track1/release
```

Comment:
Lets keep the branch name following the guidelines at https://azure.github.io/azure-sdk/policies_repobranching.html. Which means that we should name this `release/*` or `feature/*`. I assume this is only to continue to ship the older track 1 arm packages if needed. Do we expect to release them often? If not the recommendation is to create the branch on-demand to do servicing. If we do want to release them often I'd like to understand how long we expect this branch to live. If we want to keep it around until we ship stable track 2 versions then we can but I'd like to see if we can use a different name then track1 in our branches. 

For reference the python SDK has a [`feature/v3`](https://github.com/azure/azure-sdk-for-python/tree/feature/v3) branch which is for shipping their track 1 libraries that use the v3 code generator.  It would be good to follow similar pattern here.

Reply:
@weshaggard the branch will exisit until track2 is stable, and we will release them often becasue there are many release request. I will use a new branch name `feature/track1`. What do you think about it?

Reply:
I'm good with the `feature/v4` as the branch name but I suggested a more general set of branch names for the pipeline.